packages/infra/lib/api-stack/api-service.ts

```diff
@@ -380,16 +388,16 @@ export function createAPIService({
 
   // hookup autoscaling based on 90% thresholds
   const scaling = fargateService.service.autoScaleTaskCount({
-    minCapacity: isProd(props.config) ? 2 : 1,
-    maxCapacity: isProd(props.config) ? 10 : 2,
+    minCapacity: desiredTaskCount,
+    maxCapacity: maxTaskCount,
   });
   scaling.scaleOnCpuUtilization("autoscale_cpu", {
-    targetUtilizationPercent: 90,
+    targetUtilizationPercent: 80,
```

Comment:
If the [avg CPU of the service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-auto-scaling.html#:~:text=Amazon%20ECS%20publishes%20CloudWatch%20metrics%20with%20your%20service%E2%80%99s%20average%20CPU%20and%20memory%20usage) is 80%, scale out!